.eslintignore

```diff
@@ -0,0 +1,13 @@
+node_modules
+build2
+dist
+target
+test-reports
+
+packages
+scripts
+assets
+vendor
+test/unit/helpers/xhr/
```

Comment:
yes and will have lots more changes (eg. double quote -> single quote) hence I leave as it's today.
my best guess it's because those are playload copy from somewhere hence it's not really JavaScript code.

will follow up at: https://oktainc.atlassian.net/browse/OKTA-197768